src/ui/osx/TogglDesktop/test2/AppDelegate.m

```diff
@@ -1739,7 +1739,10 @@ void on_idle_notification(
 	const char *since,
 	const char *duration,
 	const int64_t started,
-	const char *description)
+	const char *description,
+    const char *project,
```

Comment:
this is a bit of a nitpick, really, but you're mixing spaces and tabs in this source file

Reply:
(and a few other ones)

Reply:
Thanks for pointing this out. Now I see that old Objective C files are using tabs and new code - spaces.
For now, updated the PR to use tabs in those files.